gpoa/frontend/appliers/control.py

```diff
@@ -25,19 +25,6 @@ class control:
     def __init__(self, name, value):
         self.control_name = name
         self.control_value = value
-        self.possible_values = self._query_control_values()
-        if self.possible_values == None:
-            raise Exception('Unable to query possible values')
-
-    def _query_control_values(self):
-        proc = subprocess.Popen(['/usr/sbin/control', self.control_name, 'list'], stdout=subprocess.PIPE)
-        for line in proc.stdout:
-            values = line.split()
-            return values
-
-    def _map_control_status(self, int_status):
```

Comment:
If **_map_control_status** will be deleted then how will works code from next pull request:
https://github.com/altlinux/gpupdate/pull/29/files ?